controllers/volumereplication_controller.go

```diff
@@ -202,13 +204,26 @@ func (r *VolumeReplicationReconciler) Reconcile(ctx context.Context, req ctrl.Re
 	}
 
 	instance.Status.LastCompletionTime = getCurrentTime()
+
 	if instance.Spec.ReplicationState == replicationv1alpha1.Resync {
-		err = r.updateReplicationStatus(instance, replicationv1alpha1.Replicating, "volume is marked for resyncing")
+		msg := "volume is marked for resyncing"
+		err = r.updateReplicationStatus(instance, replicationv1alpha1.Replicating, msg)
+		if err != nil {
+			r.Log.Error(err, "Failed to update status", "Name", instance.Name)
+			return ctrl.Result{}, err
+		}
+		r.Log.Info(msg, "Name", instance.Name)
 	} else {
-		err = r.updateReplicationStatus(instance, replicationv1alpha1.Replicating, "volume is marked "+string(instance.Spec.ReplicationState))
+		msg := "volume is marked " + string(instance.Spec.ReplicationState)
+		err = r.updateReplicationStatus(instance, replicationv1alpha1.Replicating, msg)
+		if err != nil {
+			r.Log.Error(err, "Failed to update status", "Name", instance.Name)
```

Comment:
same error is being logged inside the  `updateReplicationStatus` method.

Reply:
removed